src/not_found_handler_test.cc

```diff
@@ -0,0 +1,33 @@
+#define TEST_NOT_FOUND_HANDLER
```

Comment:
following coding style of defining the test before the actual code https://github.com/UCLA-CS130/spaceteam/blob/master/src/connection_test.cc